internal/handlers/pg/pgdb/update.go

```diff
@@ -32,21 +32,26 @@ func SetDocumentByID(ctx context.Context, tx pgx.Tx, qp *QueryParams, id any, do
 		return 0, err
 	}
 
-	table, err := getMetadata(ctx, tx, qp.DB, qp.Collection)
+	table, err := newMetadata(tx, qp.DB, qp.Collection).getTableName(ctx)
```

Comment:
Codecov complains about a missing test, an issue for it is https://github.com/FerretDB/FerretDB/issues/2149